infra/api/app-config/dev.tf

```diff
@@ -6,6 +6,8 @@ module "dev_config" {
   has_database                    = local.has_database
   database_enable_http_endpoint   = true
   has_incident_management_service = local.has_incident_management_service
+  database_max_capacity           = 2
```

Comment:
(not a blocker for merging, if its not enough, takes 2 seconds to change this number). Do we think this will be enough? I quick look at the metrics from when I ran the import job yesterday, it was trying to use ~6-7 vCPUs but only had 1 available.

Also curious if there are any other parameters we might need to tune (memory? cluster size?)

@jamesbursa I don't think we used serverless on our past project, but do you know what those databases were as far as hardware? Just looking to get a sense of scale on our usage (eg. if the dev environment in that project was the equivalent of like 16 cpus, that would be useful to know).

Reply:
> I quick look at the metrics from when I ran the import job yesterday, it was trying to use ~6-7 vCPUs but only had 1 available.

IMO we shouldn't scale up to the scale that we would need for one-time import jobs. Since they're one time, and all.

> it was trying to use ~6-7 vCPUs but only had 1 available.

This bit is interesting though! I should investigate further. I didn't know they gave us information that detailed!